src/config/settings.ts

```diff
@@ -36,7 +36,18 @@ export const ContainerEngineNames: Record<ContainerEngine, string> = {
 };
 
 export const defaultSettings = {
-  version:    CURRENT_SETTINGS_VERSION,
+  version:         CURRENT_SETTINGS_VERSION,
+  containerEngine: {
```

Comment:
This will be (temporarily) confusing since there's a _different_ `containerEngine` key. But that's understandable, so maybe we should instead focus on moving `kubernetes.containerEngine` to here instead (in a separate PR)…

(So, no action here.)

Reply:
> since there's a different containerEngine key

Yes. I initially put `imageAllowList` under `kubernetes` as well, so we could move it later when we move other unrelated keys out of `kubernetes`. But that feels like adding to our tech debt.

We will have to change the settings version, and provide migration support when we move the existing field, so I would like to keep everything as-is for now.

> (in a separate PR)

https://github.com/rancher-sandbox/rancher-desktop/issues/1223#issuecomment-1083677943 claims:
> This will be addressed as part of #1926

But we do have an openapi schema for settings now, so maybe #1926 can be closed? However, it just documents `containerEngine` in the current location, so did not address "this".